contracts/products/SolaceCoverProduct.sol

```diff
@@ -166,6 +166,7 @@ contract SolaceCoverProduct is
         _cooldownPeriod = 604800; // One-week cooldown period
         _referralReward = 50e18; // 50 DAI
         _isReferralOn = true; // Referral rewards activate
+        baseURI = "";
```

Comment:
Can you set baseURI with the following statement?
**string(abi.encodePacked("https://stats.solace.fi/policy/?chainID=", Strings.toString(block.chainid), "&policyID="));**